### PR TITLE
Added a new page called "Patching" in Installing ENT and OSS section for 1.9, 1.10 and 1.11

### PR DESCRIPTION
**Pull request by anithamuthu79**
## Description
https://jira.mesosphere.com/browse/DCOS-37738

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by anithamuthu79** [comment]
@pavisandhu @AmrAbdelrazik this is PR is ready for your review and feedback/approval.

File: pages/1.11/installing/ent/patch/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+layout: layout.pug
+navigationTitle: Patching
+title: Patching
+menuWeight: 3
+excerpt: 
+---
+
+# Patching live clusters with No downtime
+
+A DC/OS Patch describes a set of changes and supporting data designed to update, fix, or improve the features/functionality of DC/OS. A point release that consists of minor changes is also called a Patch.
```
Comment (pavisandhu) [inline pages/1.11/installing/ent/patch/index.md]: Change "Patch" to "patch" (no need for initial caps, as it's a descriptive term).

File: pages/1.11/installing/ent/patch/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+title: Patching
+menuWeight: 3
+excerpt: 
+---
+
+# Patching live clusters with No downtime
+
+A DC/OS Patch describes a set of changes and supporting data designed to update, fix, or improve the features/functionality of DC/OS. A point release that consists of minor changes is also called a Patch. 
+
+A Patching process includes the following:
```
Comment (pavisandhu) [inline pages/1.11/installing/ent/patch/index.md]: Change "A Patching process" to "A patching process"

File: pages/1.11/installing/oss/patch/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+layout: layout.pug
+navigationTitle: Patching
+title: Patching
+menuWeight: 3
+excerpt: 
+---
+
+# Patching live clusters with No downtime
+
+A DC/OS Patch describes a set of changes and supporting data designed to update, fix, or improve the features/functionality of DC/OS. A point release that consists of minor changes is also called a Patch.
```
Comment (pavisandhu) [inline pages/1.11/installing/oss/patch/index.md]: Change "A DC/OS Patch" to "A DC/OS patch"

File: pages/1.11/installing/oss/patch/index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+title: Patching
+menuWeight: 3
+excerpt: 
+---
+
+# Patching live clusters with No downtime
+
+A DC/OS Patch describes a set of changes and supporting data designed to update, fix, or improve the features/functionality of DC/OS. A point release that consists of minor changes is also called a Patch. 
+
+A Patching process includes the following:
```
Comment (pavisandhu) [inline pages/1.11/installing/oss/patch/index.md]: Change "A Patching process" to "A patching process"

Comment (anithamuthu79) [reply pages/1.11/installing/oss/patch/index.md]: Done.

File: pages/1.11/installing/oss/patch/index.md (hunk 3 above)
Comment (anithamuthu79) [reply pages/1.11/installing/oss/patch/index.md]: Done.

File: pages/1.11/installing/ent/patch/index.md (hunk 2 above)
Comment (anithamuthu79) [reply pages/1.11/installing/ent/patch/index.md]: Done.

File: pages/1.11/installing/ent/patch/index.md (hunk 1 above)
Comment (anithamuthu79) [reply pages/1.11/installing/ent/patch/index.md]: Done.

**Comment by anithamuthu79** [comment]
@klueska as per Amr's request for these pages to go live today; I'm merging this PR. Please feel free to request changes anytime. Thank You.

**Pull request merged**